src/node_url.cc

```diff
@@ -131,11 +131,13 @@ namespace url {
   static int ToUnicode(std::string* input, std::string* output) {
     output->reserve(input->length());
     *output = input->c_str();
```

Comment:
On that subject: as a slightly more wide-ranging cleanup, how about changing the return value from int to bool?  It's already effectively boolean in that it returns either 0 or -1.

Also: it looks like both the ICU and non-ICU implementations should be able to use `input->data()` instead of `input->c_str()`.  Avoids an unnecessary reallocation.